poetry.lock

```diff
@@ -1,3 +1,14 @@
+[[package]]
+name = "aiodns"
+version = "3.0.0"
+description = "Simple DNS resolver for asyncio"
+category = "main"
+optional = false
+python-versions = "*"
+
+[package.dependencies]
+pycares = ">=4.0.0"
+
```

Comment:
We don't use this, or is this a new discord.py requirement?

Reply:
It's a sub-dep of aiodns